third_party/move/move-compiler-v2/tests/checking/typing/pack_reference_2.move

```diff
@@ -0,0 +1,7 @@
+module 0x8675309::M {
```

Comment:
This test is derived from pack_reference to reveal the complete error message from V1.

Reply:
2nd error is omitted because each exp_translator has a boolean flag `had_errors` to check whether an error is already generated for it. If so, no further error is generated. The corresponding checking is [here](https://github.com/aptos-labs/aptos-core/blob/0da935350140051ffe4ad9e8893053dd90aac5a3/third_party/move/move-model/src/builder/exp_builder.rs#L374).

Reply:
After removing had_errors or putting it into `finalize_types`, duplicated errors are generated in some tests. I suggest we keep this bug open and visit it later.